omod/src/main/resources/config.xml

```diff
@@ -302,6 +302,14 @@
         </description>
     </globalProperty>
 
+    <globalProperty>
+        <property>coreapps.allowChangingVisitTime</property>
+        <defaultValue>false</defaultValue>
+        <description>
+            Be able to change the start hour of a visit.
```

Comment:
Only hour, not minute too? Oh would simply say change the start time?

Reply:
Done